### PR TITLE
Change default label expression

### PR DESCRIPTION
**Pull request by ssk2**
This changes the default label expression to be empty so that users no longer have to specify a label expression with each build. It also adds a brief upgrade note to illustrate how to upgrade existing instances.

This change depends on #19.

### CODE REVIEW

**Comment by rji** [comment]
Added some comments on #19.

File: docs/docs/upgrading.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+1. Uninstall Jenkins as per the instructions on [Getting Started]({{ site.baseurl }}/docs/). Any builds that are current in progress or queued will be lost.
+
+2. Update your local cache of the package repository: `dcos package update`.
+
+3. Install Jenkins, again following the instructions on [Getting Started]({{ site.baseurl }}/docs/). Make sure you use the same configuration file as previously, specifically pointing Jenkins to the same `host-volume`.
+
+4. Currently it is necessary to upgrade plugins by hand using the Jenkins UI at `<dcos_url>/service/jenkins/pluginManager`. In the future, it will be possible to do this through the DCOS package configuration.
+
+## Upgrading to 0.13
```
Comment (rji) [inline docs/docs/upgrading.md]: I think this should be `0.1.3`

**Comment by rji** [comment]
That one comment aside, I think this looks good. It might need a rebase though now that #19 is merged.

**Comment by ssk2** [comment]
Rebased, cheers @rji!

**Pull request merged**